app/workers/hard_worker.rb

```diff
@@ -0,0 +1,9 @@
+class HardWorker
+  include Sidekiq::Worker
+
+  def perform(current_user_email, total_users, total_articles)
```

Comment:
Be careful that workers cant receive collection as parameters. They should receive an array of ids for example and then you make a fetch in the perform